plugins/experimental/cookie_remap/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+#######################
+#
+#  Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+#  agreements.  See the NOTICE file distributed with this work for additional information regarding
+#  copyright ownership.  The ASF licenses this file to you under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with the License.  You may obtain
+#  a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software distributed under the License
+#  is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing permissions and limitations under
+#  the License.
+#
+#######################
+
+project(cookie_remap)
+
+add_atsplugin(cookie_remap
+    cookie_remap.cc
+    cookiejar.cc
+    strip.cc
+)
+
+target_include_directories(cookie_remap
+    PRIVATE
+        "${YAMLCPP_INCLUDE_DIR}"
```

Comment:
The include is fixed, but it also needs to link to the library, right?

Reply:
Cool, thanks. I couldn't find yaml-cpp::yaml-cpp eariler.